submissions/unabyband/oop_exercise/index.js

```diff
@@ -0,0 +1,122 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/unabyband/a-tiny-JS-world
+   Web app: https://unabyband.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor(species, name, gender, legs, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  printInhabitant() {
+    const nameToPrint = `<strong>${this.name}</strong>`;
+    const talkToPrint = `<em>${this.saying}</em>`;
+    const legsToPrint = `On ${this.legs} legs`;
+    const friendsToPrint = this.friends
+      ? this.friends.map((friend) => friend.name).join(", ")
+      : `${this.species} has no friends!`;
+
+    print(
+      [
+        this.species,
+        nameToPrint,
+        this.gender,
+        legsToPrint,
+        talkToPrint,
+        friendsToPrint,
+      ].join("; ")
+    );
+  }
+}
+
+class Human extends Inhabitant {
+  constructor(name, gender, saying, friends) {
+    const legs = 2;
+    const hands = 2;
+    super("human", name, gender, legs, saying, friends);
+    this.hands = hands;
+  }
+
+  //Rewriting the parent class method for printing out the string which includes unique property 'hands' for Human
+  printInhabitant() {
+    const nameToPrint = `<strong>${this.name}</strong>`;
+    const legsToPrint = `On ${this.legs} legs`;
+    const handsToPrint = `Has ${this.hands} hands`;
+    const talkToPrint = `<em>${this.saying}</em>`;
+    const friendsToPrint = this.friends.map((friend) => friend.name).join(", ");
```

Comment:
Use method overloading/overriding. This will help to make your code DRY.
Check learning materials and find out what `super`'s extended API can do for you.